pkg/model/gcemodel/autoscalinggroup.go

```diff
@@ -225,11 +225,22 @@ func (b *AutoscalingGroupModelBuilder) splitToZones(ig *kops.InstanceGroup) (map
 
 func (b *AutoscalingGroupModelBuilder) Build(c *fi.ModelBuilderContext) error {
 	for _, ig := range b.InstanceGroups {
-		instanceTemplate, err := b.buildInstanceTemplate(c, ig)
+		subnets, err := b.GatherSubnets(ig)
 		if err != nil {
 			return err
 		}
 
+		// Subnets are regional on GCE.  So it's reasonable to have one subnet per IG.
```

Comment:
I hope that this does not mean OWN subnet per IG? Subnet can be shared between many IGs?